solutions/devsprint-gabriel-goto-1/app/src/main/java/com/devpass/spaceapp/presentation/RocketDetailsActivity.kt

```diff
@@ -1,4 +1,32 @@
 package com.devpass.spaceapp.presentation
 
-class RocketDetailsActivity {
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import com.devpass.spaceapp.databinding.ActivityRocketDetailsBinding
+
+class RocketDetailsActivity : AppCompatActivity() {
```

Comment:
Como esta vai ser uma activity aberta de outro fragment, seria interessante recebermos os valores que vamos usar aqui, ja que eles vem da tela anterior 

Sabemos que vamos usar imagem, título e descrição

Para uma activity receber valores como parâmetro usamos

https://developer.android.com/guide/components/activities/parcelables-and-bundles

Fazemos assim para receber parâmetros, algo que acho interessante, no lugar de passar os valores de forma "jogada", é abstrair eles para dentro de um objeto que faça mais sentido, como um `RocketDetailArgs` por exemplo, e receber este como parameter e usar ele na nossa tela